.github/workflows/codeql.yml

```diff
@@ -2,10 +2,10 @@ name: "CodeQL"
 
 on:
   push:
-    branches: [ 'master' ]
+    # branches: [ 'master' ]
```

Comment:
Is this change related to this PR?

Reply:
i don't believe it was, have resolved it now